#PresenceDetection.msg

# State of the presence detection
uint8 TARGET_MOVING = 0
uint8 TARGET_STANDING = 1

uint8 target_state # The state of the target, should have one of the values above


uint8 TARGET_OUT_OF_RANGE = 0
uint8 TARGET_IN_RANGE = 1 

uint8 presence_state  # The state of the presence detection, should have one of the values above




